### ISSUE TITLE
what is the meaning conv2D model output shape if output is as under

### ISSUE THREAD

**Issue by abdulqayyum** [fchollet/keras#4106]
Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by abdulqayyum** [comment]
here is the summary of the ongoing model. 
Question 1: What is Output Shape? I think for (None,3,224,64): None =?, 3=may be channels but they are decreasing to negative values, 224=dimention of my input but do not know if it is height or width?, 64 is the number of filters. The file can be accessed [here](Test-CNN/colorize.ipynb)
Layer (type)                     Output Shape          Param #     Connected to                     
====================================================================================================
convolution2d_15 (Convolution2D) (None, 3, 224, 64)    129088      convolution2d_input_3[0][0]      
____________________________________________________________________________________________________
activation_21 (Activation)       (None, 3, 224, 64)    0           convolution2d_15[0][0]           
____________________________________________________________________________________________________
convolution2d_16 (Convolution2D) (None, 1, 111, 64)    36928       activation_21[0][0]              
____________________________________________________________________________________________________
activation_22 (Activation)       (None, 1, 111, 64)    0           convolution2d_16[0][0]           
____________________________________________________________________________________________________
batchnormalization_9 (BatchNormal(None, 1, 111, 64)    2           activation_22[0][0]              
____________________________________________________________________________________________________
convolution2d_17 (Convolution2D) (None, 1, 111, 128)   73856       batchnormalization_9[0][0]       
____________________________________________________________________________________________________
activation_23 (Activation)       (None, 1, 111, 128)   0           convolution2d_17[0][0]           
____________________________________________________________________________________________________
convolution2d_18 (Convolution2D) (None, 0, 55, 128)    147584      activation_23[0][0]              
____________________________________________________________________________________________________
activation_24 (Activation)       (None, 0, 55, 128)    0           convolution2d_18[0][0]           
____________________________________________________________________________________________________
batchnormalization_10 (BatchNorma(None, 0, 55, 128)    256         activation_24[0][0]              
____________________________________________________________________________________________________
convolution2d_19 (Convolution2D) (None, 0, 55, 256)    295168      batchnormalization_10[0][0]      
____________________________________________________________________________________________________
activation_25 (Activation)       (None, 0, 55, 256)    0           convolution2d_19[0][0]           
____________________________________________________________________________________________________
convolution2d_20 (Convolution2D) (None, -2, 53, 256)   590080      activation_25[0][0]              
____________________________________________________________________________________________________
